include/rtc/websocket.hpp

```diff
@@ -46,6 +46,8 @@ class RTC_CPP_EXPORT WebSocket final : private CheshireCat<impl::WebSocket>, pub
 		bool disableTlsVerification = false; // if true, don't verify the TLS certificate
 		optional<ProxyServer> proxyServer;   // unsupported for now
 		std::vector<string> protocols;
+		int maxMissedPongsAllowed = 2;                   // -1 is no check
```

Comment:
Actually, it could be good to rename the setting to something with 0 as the default value in order to expose the same setting in the C API (the default value has to be 0 there). 

I suggest `maxOutstandingPings`, with 1 meaning 0 missed pongs, so 0 can mean disabled in the C API (the phrasing also sounds more common). What do you think?

Reply:
ok, great idea

so just to be clear, you still want the optional, right? so it becomse `optional<int> maxOutstandingPings`

Reply:
Yes, I'd prefer making it optional for clarity and consistency with other configuration settings in (`rtc::Configuration` for instance).